packages/next/site/README.md

```diff
@@ -0,0 +1,25 @@
+# Patternfly React Docs
+This site uses gatsby.
+
+For an overview of the project structure please refer to the [Gatsby documentation - Building with Components](https://www.gatsbyjs.org/docs/building-with-components/).
+
+
+
+### Note: All commands below assume you are on the root directory in this repository.
+## Install
+Run to install all the dependencies
+```sh
+lerna bootstrap
+```
+
+## Running
+To start the site locally.
+```sh
+yarn start:site
+```
+
+## Building
+To build the site.
+```sh
+yarn build:site
```

Comment:
Possibly.  I wanted to get this up and get some eyes on it.  I might be able to figure that out before this goes in.  If not, it will be a quick follow up.

Reply:
Netlify has been hooked up and will build once this is merged.  After that PR's will be built as well.